packages/docusaurus-theme-classic/codeTranslations/fil.json

```
@@ -0,0 +1,55 @@
+  "theme.PwaReloadPopup.closeButtonAriaLabel": "Isara",
+  "theme.PwaReloadPopup.info": "May available na bagong bersiyón",
+  "theme.PwaReloadPopup.refreshButtonText": "I-refresh",
+  "theme.SearchBar.label": "Maghanap",
+  "theme.SearchPage.algoliaLabel": "Paghahanap hatid ng Algolia",
+  "theme.SearchPage.documentsFound.plurals": "Isang dokumento ang nahanap|{count} na mga dokumento ang nahanap",
+  "theme.SearchPage.emptyResultsTitle": "Maghanap sa dokumentasyón",
+  "theme.SearchPage.existingResultsTitle": "Resulta ng paghahanap para sa \"{query}\"",
+  "theme.SearchPage.fetchingNewResults": "Kumukuha ng mga bagong resulta...",
+  "theme.SearchPage.inputLabel": "Maghanap",
+  "theme.SearchPage.inputPlaceholder": "I-type and inyong hinahanap dito",
+  "theme.SearchPage.noResultsText": "Walang resultang nahanap",
+  "theme.blog.paginator.navAriaLabel": "Nabegasyón para sa pahina na listahan ng blog",
+  "theme.blog.paginator.newerEntries": "Mas bagong mga éntri",
+  "theme.blog.paginator.olderEntries": "Mas lumang mga éntri",
+  "theme.blog.post.paginator.navAriaLabel": "Nabegasyón para sa page ng blog post",
+  "theme.blog.post.paginator.newerPost": "Mas Bagong Post",
+  "theme.blog.post.paginator.olderPost": "Mas Lumang Post",
+  "theme.blog.post.plurals": "Isang post|{count} na mga post",
+  "theme.blog.post.readMore": "Basahin Pa",
+  "theme.blog.post.readingTime.plurals": "Isang minutong basahin|{readingTime} minutong basahin",
+  "theme.blog.tagTitle": "{nPosts} na may tag na \"{tagName}\"",
+  "theme.common.editThisPage": "I-edit ang page",
+  "theme.common.headingLinkTitle": "Direktang link patungo sa heading",
+  "theme.common.skipToMainContent": "Lumaktaw patungo sa pangunahing content",
+  "theme.docs.paginator.navAriaLabel": "Nabegasyón para sa mga pahinang docs.",
+  "theme.docs.paginator.next": "Sumunod",
+  "theme.docs.paginator.previous": "Naraaan",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "Itupî ang sidebar",
+  "theme.docs.sidebar.collapseButtonTitle": "Itupî ang sidebar",
+  "theme.docs.sidebar.expandButtonAriaLabel": "Palakihin ang sidebar",
+  "theme.docs.sidebar.expandButtonTitle": "Palakihin ang sidebar",
+  "theme.docs.sidebar.responsiveCloseButtonLabel": "Isara ang menu",
+  "theme.docs.sidebar.responsiveOpenButtonLabel": "Buksan ang menu",
+  "theme.docs.versions.latestVersionLinkLabel": "pinakahuling bersiyón",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Para sa up-to-date na dokumentasyón, tingnan ang {latestVersionLink} ({versionLabel}).",
+  "theme.docs.versions.unmaintainedVersionLabel": "Ito ay dokumentasyón para sa {siteTitle} {versionLabel} na hindi na aktibong mine-maintain.",
+  "theme.docs.versions.unreleasedVersionLabel": "Ito ay hindi pa nailalabas na dokumentasyón para sa bersiyón na {siteTitle} {versionLabel}.",
+  "theme.lastUpdated.atDate": " noong {date}",
+  "theme.lastUpdated.byUser": " ni {user}",
+  "theme.lastUpdated.lastUpdatedAtBy": "Huling inapdeyt {atDate}{byUser}",
```

```suggestion
  "theme.lastUpdated.lastUpdatedAtBy": "Huling inapdeyt{atDate}{byUser}",
```

Omitting the space is on purpose here, or it would lead. to a double-space